ibmcloud/README.md

```diff
@@ -255,82 +255,112 @@ $ install cloud-api-adaptor /usr/local/bin/
 
 ## Launch Cloud API adaptor
 
-You can start Cloud API adaptor as follows on the worker node. Please update the variable values if you use custom ones. The VPC, region, zone, subnet, security name are must same as the values you just used for creating VPC.
+A terraform template that will start the `cloud-api-adaptor` process on the Kubernetes worker node is available in [ibmcloud/terraform/start-cloud-api-adaptor](./terraform/start-cloud-api-adaptor).
+
+Create a `terraform.tfvars` file in the [template directory](./terraform/start-cloud-api-adaptor) for this Terraform template on your `development machine`. The `terraform.tfvars` file should look like this
 
-```bash
-$ api_key=<your API key>
-$ image_name=<pod VM image name>
-$ ssh_key_name=<your SSH key name>
-$ vpc_name=tok-vpc
-$ subnet_name=tok-primary-subnet
-$ security_group_name=tok-primary-security-group
-$ vpc_region=jp-tok
-$ vpc_zone=jp-tok-2
-$ instance_profile=bx2-2x8
 ```
-> **Tip:**: Modify **instance_profile** to change the type of VSI provisioned e.g. to create 2 vCPU, 8GB RAM balanced `s390x` VSIs for the peer-pod use `instance_profile=bz2-2x8`
+ibmcloud_api_key = "<your API Key>"
+cluster_name = "<cluster name>"
+ssh_key_name = "<your SSH key name>"
```

Comment:
it ask me provide value for `podvm_image_name`, so that it's better we add it here?
```
podvm_image_name = "<name of your pod VM image>"
```

Reply:
Yes, added that because `podvm_image_name` has no default value in variables.tf.